### ISSUE TITLE
CSS properties and vendor prefixes

### ISSUE THREAD

**Issue by le717** [Brickimedia/brickimedia#187]
I'm opening this here because I'm seeing this on most all your themes.

Your vendor prefixed CSS properties are either out-of-date, in the wrong order, or some other error. Let me explain using your Refreshed theme.

First off, [lines 28 - 31 in main.css](https://github.com/Brickimedia/Refreshed/blob/master/refreshed/main.css#L28-L31). You have all the vendor prefixed versions for `linear-gradient`, except for `linear-gradient` itself. I happen to know the `-moz-` prefix is not needed. In that case, Firefox will not render the gradient. You need a non-prefixed copy in that rule.

Quickly switching to your Lia theme.

[Lines 1090 - 1094 in lia.css](https://github.com/Brickimedia/Lia/blob/master/lia/lia.css#L1090-1094). The non-prefixed version [always](http://css-tricks.com/ordering-css3-properties/) goes at the bottom _after_ the prefixes. 

Basically, your CSS has a lot of issues, and you need to go ahead and start validating it now and not later on. It looks like you'll have a lot of work to perform to fix it all. 

Finally, I'd like to recommend use of the [-prefix-free](http://leaverou.github.io/prefixfree/) library. You only have to write the standard property, and it adds the prefixes as needed. I use it myself at http://le717.github.io/, and it works [quite well](http://le717.github.io/css/style.css). :smiley:
